addons/mail/static/src/components/attachment_viewer/attachment_viewer.js

```diff
@@ -93,13 +93,13 @@ export class AttachmentViewer extends Component {
      * @private
      */
     _handleImageLoad() {
-        if (!this.attachmentViewer.exists() || !this.attachmentViewer.attachment) {
+        if (!this.attachmentViewer.exists() || !this.attachmentViewer.attachmentViewerViewable) {
             return;
         }
         const refs = this._getRefs();
-        const image = refs[`image_${this.attachmentViewer.attachment.id}`];
+        const image = refs[`image_${this.attachmentViewer.attachmentViewerViewable.id}`];
```

Comment:
I believe this is no longer correct, `id` of attachment could conflict with `id` of document but we would need separate `ref` in that case.

Reply:
consider using `localId` instead of `id` for the `ref` instead

Reply:
It's just a unique identifier we have for each record. It's currently based on model name / identifying fields, but this is a technical choice to help debug, but this could as well be a unique hash.